app/scripts/modules/core/serverGroup/serverGroup.write.service.js

```diff
@@ -60,6 +60,23 @@ module.exports = angular
       });
     }
 
+    function rollbackServerGroup(serverGroup, application, params={}) {
+      params.asgName = serverGroup.name;
+      params.serverGroupName = serverGroup.name;
```

Comment:
Do we need both here? You just wrote the Orca functionality - does it expect the "asgName" field to be populated from the client?